static/app/components/demoSandboxButton.tsx

```diff
@@ -0,0 +1,94 @@
+import Button from 'sentry/components/button';
+import {Organization} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  /**
+   * The deep link scenario
+   */
+  scenario:
+    | 'performance'
+    | 'releases'
+    | 'alerts'
+    | 'discover'
+    | 'dashboards'
+    | 'projects'
+    | 'oneDiscoverQuery'
+    | 'oneIssue'
+    | 'oneBreadcrumb'
+    | 'oneStackTrace'
+    | 'oneTransaction'
+    | 'oneWebVitals'
+    | 'oneTransactionSummary'
+    | 'oneRelease';
+  organization: Organization;
+  buttonText: string;
+  /**
+   * Which project we should link to in the sandbox
+   */
+  projectSlug?: 'react' | 'python' | 'ios' | 'android' | 'react-native';
+  /**
+   * Matching on the error type or title
+   */
+  errorType?: string;
+
+  buttonProps?: Partial<React.ComponentProps<typeof Button>>;
+
+  clientData?: SandboxData;
+};
+
+export interface SandboxData {
+  skipEmail?: boolean;
+  acceptedTracking?: boolean;
+  cta?: {
+    title: string;
+    shortTitle: string;
+    url: string;
+  };
+}
+
+// Renders a form that will kick off the sandbox around children
+// which should include be a button. If the sandbox is hidden,
+// don't render the children
+function DemoSandboxButton({
```

Comment:
Want to use a docblock style comment here so we get the language-server automatic documentation on hover?